csrc/fusion_profiler.cpp

```diff
@@ -43,7 +43,14 @@ void record_cupti_activity(CUpti_Activity* pRecord, FILE* pFileHandle) {
 
       KernelProfile prof;
       prof.name.assign(demangle(pKARecord->name));
-      size_t start = prof.name.find("kernel");
+
+      size_t kernel_start = prof.name.find("kernel");
+      size_t nvfuser_start = prof.name.find("nvfuser");
+      size_t start = std::min(kernel_start, nvfuser_start);
```

Comment:
nitpick. The `min` here felt a bit messy, maybe we can explicitly check each `find` result != npos?!